console/src/components/Services/Data/TablePermissions/Permissions.js

```diff
@@ -236,39 +245,51 @@ class Permissions extends Component {
     }
     return Promise.resolve();
   }
-  deleteSetKeyVal(e) {
+  checkUpdatePresetVer(version) {
+    let showUpdatePresets = false;
+    try {
+      showUpdatePresets = semverCheck('permUpdatePresets', version);
```

Comment:
semverCheck returns either true or false. I think in the previous version of semverCheck which was throwing an exception when the version is not in correct format.

Is it possible to change this ?